lnrpc/peersrpc/peers.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package peersrpc;
```

Comment:
Cool, this can eventually be used for things like using a single node to inject certain gossip messages updates, which is related to this issue: #6294